.vitepress/config.ts

```diff
@@ -696,6 +696,10 @@ function sidebarHome() {
               text: "Querying the Blobstream proofs",
               link: "/developers/blobstream-proof-queries",
             },
+            {
+              text: "Example implementation of Blobstream by CryptoKass",
```

Comment:
Should ask for his permission first?

Reply:
@CryptoKass do you mind if we add this? do yo have any suggested changes for this?

Reply:
This looks good 👍